src/test/groovy/graphql/execution/ExecutionParametersTest.java

```diff
@@ -0,0 +1,41 @@
+package graphql.execution;
+
+import graphql.language.Field;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Map;
+
+import static graphql.Scalars.*;
+import static graphql.execution.ExecutionParameters.newParameters;
+import static graphql.execution.TypeInfo.newTypeInfo;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonMap;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotSame;
+import static org.junit.Assert.assertSame;
+
+public class ExecutionParametersTest {
```

Comment:
Please have a look here for how to use groovy and java 8 interfaces: https://github.com/graphql-java/graphql-java/blob/a0096cb2f95e3d471d3922de9d9389c56c518c54/src/test/groovy/graphql/schema/idl/SchemaPrinterTest.groovy#L22